assets/multiqc_config.yml

```diff
@@ -89,6 +89,9 @@ sp:
     fn: "*_fastqc.zip"
   nonpareil:
     fn: "nonpareil_all_samples.json"
+  filtlong:
+    contents: Scoring long reads
+    contents_re: " "
```

Comment:
I think you still need the number of lines to read below this one to speed up detection

Reply:
The original definition from the MultiQC module will be used. And keeping the `contents: Scoring long reads` isn't necessary, but I thought it might make it easier to understand.